src/blog/_single.html

```diff
@@ -29,11 +29,11 @@
 {% endblock %}
 
 {% block content_sidebar %}
-    <section class="block block__flush-top">
+    <section class="block block__flush-top" >
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/f18474a240fd0f56bdd225730736293f2f28c11c